crates/collateral/src/lib.rs

```diff
@@ -165,6 +165,29 @@ impl<T: Config> Module<T> {
         Ok(())
     }
 
+    /// Transfer DOT from a sender a receiver. Can only fail if
+    /// the sender account has too little locked DOT.
+    ///
+    /// # Arguments
+    ///
+    /// * `sender` - the account to take DOT from
+    /// * `receiver` - the receiver of the DOT
+    /// * `amount` - the to be transferred amount
+    /// * `status` - the status of the amount, once transferred (`reserved` or `free`)
+    pub fn repatriate_reserved(
+        sender: T::AccountId,
+        receiver: T::AccountId,
+        amount: BalanceOf<T>,
+        status: BalanceStatus,
```

Comment:
Do we really need to expose this parameter?

Reply:
The `status` parameter? Since we're exposing the `repatriate_reserved` function, we should do it in a reusable way - thus exposing all of its parameters